### PR TITLE
Dantooine  Electric Boogaloo

### PR DESCRIPTION
**Pull request by Scorchys12**
Dantooine Planet update
Bible design updated with the following: Dantooine World creatures, 4 food two recipes. Bol rifle and its blueprint, carrot, bol leather, and milk crafting items and fishing locations.
https://docs.google.com/spreadsheets/d/1QrqYFbq3cuAmR9jzTCAlrG8bXtaemxhEHF4DrX-i1t8/edit#gid=207006097

### CODE REVIEW

File: SWLOR.Game.Server/Enumeration/PlanetType.cs [hunk 1]
```diff
@@ -70,6 +70,13 @@ public enum PlanetType
             800,
             true)]
         Dathomir = 64,
+        [Planet("Dantooine",
+    "Dantooine - ",
```
Comment (zunath) [inline SWLOR.Game.Server/Enumeration/PlanetType.cs:74]: Please fix the tabbing here. It should be lined up similar to the other planet entries.

**Comment by zunath** [comment]
+20 Accuracy & Force Attack for commonly acquired ingredients is too high. Either reduce the amount it increases and reduce its level or change the ingredients needed to use more difficult to obtain items.

File: SWLOR.Game.Server/Feature/RecipeDefinition/SmitheryRecipeDefinition/RangedRecipes.cs [hunk 2]
```diff
@@ -277,6 +277,24 @@ private void Rifles()
                 .Component("ref_scordspar", 5)
                 .Component("ref_plagionite", 5)
                 .Component("ref_keromber", 5);
+
+            // Bol Rifle
```
Comment (zunath) [inline SWLOR.Game.Server/Feature/RecipeDefinition/SmitheryRecipeDefinition/RangedRecipes.cs:281]: This is stronger than even the Chiro Rifle. If it's supposed to be on par with the Chiro then please reduce its DMG and give it equivalent additional stats.

File: Module/uti/dan_heavy_armor.uti.json [hunk 3]
```diff
@@ -0,0 +1,298 @@
+{
```
Comment (zunath) [inline Module/uti/dan_heavy_armor.uti.json:1]: Please mark this item as identified.

File: Module/uti/dan_hyper_gloves.uti.json [hunk 4]
```diff
@@ -0,0 +1,264 @@
+{
```
Comment (zunath) [inline Module/uti/dan_hyper_gloves.uti.json:1]: Please mark this item as identified.

File: Module/uti/dan_kin_boots.uti.json [hunk 5]
```diff
@@ -0,0 +1,210 @@
+{
```
Comment (zunath) [inline Module/uti/dan_kin_boots.uti.json:1]: The name of the item is "Boots" but the perk needed is "Leggings". Please correct so it's consistently named.

File: Module/uti/dan_spear.uti.json [hunk 6]
```diff
@@ -2,23 +2,23 @@
   "__data_type": "UTI ",
```
Comment (zunath) [inline Module/uti/dan_spear.uti.json:2]: 55 DMG is obnoxiously high. Please reduce.

File: Module/uti/krafters_kebab.uti.json [hunk 7]
```diff
@@ -0,0 +1,204 @@
+{
+  "__data_type": "UTI ",
```
Comment (zunath) [inline Module/uti/krafters_kebab.uti.json:2]: Please set the identified flag to true under properties.

File: Module/uti/map_072.uti.json [hunk 8]
```diff
@@ -0,0 +1,129 @@
+{
```
Comment (zunath) [inline Module/uti/map_072.uti.json:1]: Please set the identified property to true under properties.

File: Module/uti/map_073.uti.json [hunk 9]
```diff
@@ -0,0 +1,129 @@
+{
```
Comment (zunath) [inline Module/uti/map_073.uti.json:1]: Please set the identified property to true under properties.

File: Module/uti/map_074.uti.json [hunk 10]
```diff
@@ -0,0 +1,131 @@
+{
```
Comment (zunath) [inline Module/uti/map_074.uti.json:1]: Please set the identified property to true under properties.

File: Module/uti/map_075.uti.json [hunk 11]
```diff
@@ -0,0 +1,111 @@
+{
```
Comment (zunath) [inline Module/uti/map_075.uti.json:1]: Please set the identified property to true under properties.

File: Module/uti/map_076.uti.json [hunk 12]
```diff
@@ -0,0 +1,131 @@
+{
```
Comment (zunath) [inline Module/uti/map_076.uti.json:1]: Please set the identified property to true under properties.

File: Module/uti/map_077.uti.json [hunk 13]
```diff
@@ -0,0 +1,131 @@
+{
```
Comment (zunath) [inline Module/uti/map_077.uti.json:1]: Please set the identified property to true under properties.

File: Module/uti/map_078.uti.json [hunk 14]
```diff
@@ -0,0 +1,131 @@
+{
```
Comment (zunath) [inline Module/uti/map_078.uti.json:1]: Please set the identified property to true under properties.

File: Module/uti/map_70.uti.json [hunk 15]
```diff
@@ -0,0 +1,129 @@
+{
```
Comment (zunath) [inline Module/uti/map_70.uti.json:1]: Please set the identified property to true under properties.

File: Module/uti/map_70.uti.json [hunk 16]
```diff
@@ -0,0 +1,129 @@
...
+  "__data_type": "UTI ",
```
Comment (zunath) [inline Module/uti/map_70.uti.json:2]: Please correct the resref on this item. It's missing the leading zero

File: Module/uti/map_71.uti.json [hunk 17]
```diff
@@ -0,0 +1,129 @@
+{
```
Comment (zunath) [inline Module/uti/map_71.uti.json:1]: Please set the identified property to true under properties.

File: Module/uti/map_71.uti.json [hunk 18]
```diff
@@ -0,0 +1,129 @@
...
+  "__data_type": "UTI ",
```
Comment (zunath) [inline Module/uti/map_71.uti.json:2]: Please correct the resref on this item. It is missing the leading zero.

File: Module/uti/munch_fungusb.uti.json [hunk 19]
```diff
@@ -0,0 +1,171 @@
+{
```
Comment (zunath) [inline Module/uti/munch_fungusb.uti.json:1]: Please set the identified property to true under properties.

File: Module/uti/recipe_bolrifle.uti.json [hunk 20]
```diff
@@ -0,0 +1,129 @@
...
+    "value": {}
+  },
+  "Identified": {
+    "type": "byte",
+    "value": 0
+  },
+  "LocalizedName": {
+    "type": "cexolocstring",
+    "value": {
+      "0": "Blueprint: Bol Rilfe"
```
Comment (zunath) [inline Module/uti/recipe_bolrifle.uti.json:42]: Typo here. Rilfe -> Rifle

File: Module/are/dan_junglemount.are.json [hunk 21]
```diff
@@ -0,0 +1,5334 @@
...
+    "value": 1519896
+  },
+  "MoonShadows": {
+    "type": "byte",
+    "value": 0
+  },
+  "Name": {
+    "type": "cexolocstring",
+    "value": {
+      "0": "Dantooine -  Jungle Mountain"
```
Comment (zunath) [inline Module/are/dan_junglemount.are.json:90]: Typo: extra space between "-" and "Jungle"

File: Module/are/dan_warehouse.are.json [hunk 22]
```diff
@@ -0,0 +1,2926 @@
...
+    "value": 0
+  },
+  "MoonShadows": {
+    "type": "byte",
+    "value": 0
+  },
+  "Name": {
+    "type": "cexolocstring",
+    "value": {
+      "0": "Dantooine - Abandon Warehouse"
```
Comment (zunath) [inline Module/are/dan_warehouse.are.json:90]: Typo: Should be "Abandoned"

File: Module/utw/fp_danmount.utw.json [hunk 23]
```diff
@@ -0,0 +1,51 @@
+{
```
Comment (zunath) [inline Module/utw/fp_danmount.utw.json:1]: Please change this resref to be consistent with the others. If the numbering is right, this should be "fp_035"

File: SWLOR.Game.Server/Service/AchievementService/AchievementType.cs [hunk 24]
```diff
@@ -299,6 +299,33 @@ public enum AchievementType
 
         [Achievement("The Legendary Rod", "Completed the quest 'The Legendary Rod'.", true)]
         TheLegendaryRod = 110,
+
+        [Achievement("Explore Dantooine Orbit", "Explore Dantooine's Orbit'.", true)]
```
Comment (zunath) [inline SWLOR.Game.Server/Service/AchievementService/AchievementType.cs:303]: Typo: extra ' after Orbit.

File: SWLOR.Game.Server/Service/AchievementService/AchievementType.cs [hunk 25]
```diff
@@ -299,6 +299,33 @@ public enum AchievementType
...
+        [Achievement("Explore Dantooine Field", "Explore Dantooine's Fields.", true)]
+        ExploreDantooineField = 112,
+
+        [Achievement("Explore Dantooine Colony", "Explore Dantooine's Colony.", true)]
+        ExploreDantooineColony = 113,
+
+        [Achievement("Explore Dantooine Lake", "Explore Dantooine's Lake.", true)]
+        ExploreDantooineLake = 114,
+        
+        [Achievement("Explore Dantooine Kinrath Cave", "Explore Dantooine's Kinrath Tunnels.", true)]
```
Comment (zunath) [inline SWLOR.Game.Server/Service/AchievementService/AchievementType.cs:315]: Minor issue: Please make it consistent. Is it Caves or Tunnels?

File: SWLOR.Game.Server/Service/AchievementService/AchievementType.cs [hunk 26]
```diff
@@ -299,6 +299,33 @@ public enum AchievementType
...
+        ExploreDantooineKinrathTunnel = 115,
+
+        [Achievement("Explore Dantooine Wild Plains", "Explore Dantooine's Wild Plains.", true)]
+        ExploreDantooineWildPlains = 116,
+
+        [Achievement("Explore Dantooine South Plains", "Explore Dantooine's Tribe.", true)]
```
Comment (zunath) [inline SWLOR.Game.Server/Service/AchievementService/AchievementType.cs:321]: Description doesn't match the area. Please correct

File: SWLOR.Game.Server/Service/AchievementService/AchievementType.cs [hunk 27]
```diff
@@ -299,6 +299,33 @@ public enum AchievementType
...
+        ExploreDantooineTribe = 117,
+
+        [Achievement("Explore Dantooine Abandon Warehouse", "Explore Dantooine's Abandon Base.", true)]
```
Comment (zunath) [inline SWLOR.Game.Server/Service/AchievementService/AchievementType.cs:324]: Abandon -> Abandoned. Also, consistency here too - is it a Warehouse or a Base? Please update to reflect appropriately.

File: SWLOR.Game.Server/Service/AchievementService/AchievementType.cs [hunk 28]
```diff
@@ -299,6 +299,33 @@ public enum AchievementType
...
+        ExploreDantooineAbandonWare = 118,
```
Comment (zunath) [inline SWLOR.Game.Server/Service/AchievementService/AchievementType.cs:325]: Please correct the enum name to match too. And fully spell out words, please

File: SWLOR.Game.Server/Service/KeyItemService/KeyItemType.cs [hunk 29]
```diff
@@ -151,6 +151,25 @@ public enum KeyItemType
...
+        DantooineKinrathMap = 72,
+        [KeyItem(KeyItemCategoryType.Maps, "Dantooine - Tribal Map", true, "Map of the South Plains Tribes on Dantooine.")]
+        DantooineTribalMap = 73,
+        [KeyItem(KeyItemCategoryType.Maps, "Dantooine - Forsaken Jungles Map", true, "Map of the Forsaken Jungles Caverns on Dantooine.")]
+        DantooineForsakenJungleMap = 74,
+        [KeyItem(KeyItemCategoryType.Maps, "Dantooine - Mountain Jungles Map", true, "Map of the Mountain Jungles on Dantooine.")]
+        DantooineMountianJunglesMap = 75,
+        [KeyItem(KeyItemCategoryType.Maps, "Dantooine - Crystal Cave Map", true, "Map of the Crystal Cave on Dantooine.")]
+        DantooineCrystalCaveMap = 76,
+        [KeyItem(KeyItemCategoryType.Maps, "Dantooine - Abandon Warehouse  Map", true, "Map of the Abandon Warehouse Base on Dantooine.")]
```
Comment (zunath) [inline SWLOR.Game.Server/Service/KeyItemService/KeyItemType.cs:169]: Typo: Abandon -> Abandoned

Comment (zunath) [inline SWLOR.Game.Server/Service/KeyItemService/KeyItemType.cs:169]: You also have an extra space after "Warehouse" which should be removed.

File: SWLOR.Game.Server/Service/KeyItemService/KeyItemType.cs [hunk 30]
```diff
@@ -151,6 +151,25 @@ public enum KeyItemType
...
+        DantooineWareHouseMap = 77,
```
Comment (zunath) [inline SWLOR.Game.Server/Service/KeyItemService/KeyItemType.cs:170]: Typo: WareHouse -> Warehouse

File: SWLOR.Game.Server/Service/KeyItemService/KeyItemType.cs [hunk 31]
```diff
@@ -151,6 +151,25 @@ public enum KeyItemType
...
+        [KeyItem(KeyItemCategoryType.Maps, "Dantooine - Canyon River Map", true, "Map of the Canyon Rivers on Dantooine.")]
+        DantooineCanyonRivereMap = 78,
```
Comment (zunath) [inline SWLOR.Game.Server/Service/KeyItemService/KeyItemType.cs:172]: Typo: Rivere -> River

**Review by zunath — CHANGES_REQUESTED** [review]
Changes requested

File: Module/uti/krafters_kebab.uti.json [hunk 32]
```diff
@@ -0,0 +1,204 @@
+{
```
[the remark this answers is not in the document]
Comment (Scorchys12) [reply Module/uti/krafters_kebab.uti.json:1]: i reduced it down to 7+ to all, though unsure if that's too low with the ingredients required.

**Comment by zunath** [comment]
New PR submitted due to lack of permissions to push changes to this one. https://github.com/zunath/SWLOR_NWN/pull/1717

**Pull request closed without merging**